libs/blocks/table/table.js

```diff
@@ -287,6 +287,23 @@ function applyStylesBasedOnScreenSize(table, originTable) {
   const isMerch = table.classList.contains('merch');
   const deviceBySize = defineDeviceByScreenSize();
 
+  const setRowStyle = () => {
+    const sectionRow = Array.from(table.getElementsByClassName('section-row'));
+    if (sectionRow.length > 0) {
+      const colsForTablet = sectionRow[0].children.length - 1;
+      const percentage = 100 / colsForTablet;
+      const templateColumnsValue = `repeat(auto-fit, ${percentage}%)`;
+      sectionRow.forEach((row) => {
+        if (isMerch) return;
```

Comment:
Since `isMerch` is declared on line 287, if that is `true` then every single row will be skipped right?

If that's the case then can you early return right at the beginning of `setRowStyle`?

Reply:
I have updated the code